src/pages/ProfilesExplorerView/domain/variables/ServiceNameVariable/ServiceNameVariable.tsx

```diff
@@ -117,8 +118,8 @@ export class ServiceNameVariable extends QueryVariable {
       <Cascader
         // we add a key to ensure that the Cascader selects the initial value properly when landing on the page
         // and when switching exploration types, because the value might also be changed after the component has been rendered by SceneProfilesExplorer
-        // (e.g. in SceneExploreServiceProfileTypes)
-        key={String(loading) + String(value)}
+        // (e.g. in SceneExploreServiceProfileTypes, also when clicking on an Investigation link, it ensures the value is properly updated)
+        key={nanoid(5)}
```

Comment:
Wouldn't `value` change when link is clicked from Investigations causing the key to change? 🤔